### ISSUE TITLE
ability to change seekbar mouseover tooltip font/style

### ISSUE THREAD

**Issue by videonerd** [mpv-player/mpv#5809]
All fonts/style can be changed except the seekbar mouseover tooltip. It would be good to be able to change this. Or am I missing something?

**Comment by bitingsock** [comment]
It uses the --osd-font setting.

**Comment by videonerd** [comment]
@bitingsock Nope it doesn't, just tried changing osd font settings which has no impact on the mouseover tooltip (tried osd-border-size, osd-blur, etc nothing has any impact).

**Comment by bitingsock** [comment]
To be clear you are talking about the On-Screen-Controller, right? Hovering over the seekbar and seeing the timecode?

**Comment by videonerd** [comment]
@bitingsock yes that is correct - i note that the osd styling doesn't effect it as suggested. Any idea? Thank you.

**Comment by bitingsock** [comment]
Just confirmed with a zero-config: this setting changes it for me. what is your version?

**Comment by videonerd** [comment]
@bitingsock upon further investigation, my observations are as follows - this specific timecode on mouseover the OSC only respects osd-font but doesn't respect any other setting e.g. osd-border-size, 
osd-shadow-color, osd-shadow-offset, osd-color etc. It only defaults to white text with black shadow/blur.

Hope this provides more information.

**Comment by bitingsock** [comment]
Looking at the osc code https://github.com/mpv-player/mpv/blob/master/player/lua/osc.lua we can see that it hardcodes all those other values. Look at line 74 for example for the "timecode" style. you would have to change it there to get what you want.
